pkg/controller/healthcheck/general/deployment.go

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) 2019 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package general
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/gardener/gardener-extensions/pkg/controller/healthcheck"
+
+	"github.com/gardener/gardener/pkg/utils/kubernetes/health"
+	"github.com/go-logr/logr"
+	v1 "k8s.io/api/apps/v1"
```

Comment:
```suggestion
	appsv1 "k8s.io/api/apps/v1"
```

to improve the reading. `v1` could be `k8s.io/api/apps/v1`, `k8s.io/api/core/v1` or `k8s.io/apimachinery/pkg/apis/meta/v1`.